source/crud/read.txt

```diff
@@ -0,0 +1,166 @@
+.. _mongosh-read:
+
+===============
+Query Documents
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+Use the :method:`db.collection.find()` method in the |mdb-shell|
+to query documents in a collection.
+
+.. include:: /includes/fact-sample-data-examples.rst
+
+.. TODO: Make a callout here to the Connect to an Atlas Cluster section
+   in the connection docs
+
+Select All Documents in a Collection
```

Comment:
Is this section titled this way because of the relationship to the SQL operation? Seems like we use `query` elsewhere.